docs/USAGE.md

```diff
@@ -68,14 +59,51 @@ The component's style can be customized like this:
     );
 }
 ```
-## `wix-style-processor`
-TPA's get their colors and fonts (most of them) from the template's palette, 
-and let the users override them using the settings panel.
+
+#### An optimized way for style overriding - New!
+Starting of version 2.10.0, new components will support a more optimized way to override styles, which **reduces bundle size significantly**.
+
+*Note*: This will become the official way to override styles, however not all components are compatible yet (please check each component's documentation page to see if it supports the new way).
+
+```css
+/* CheckoutButton.st.css */
+:import {
+    -st-from: "wix-ui-tpa/dist/src/components/Button/Button.st.css";
```

Comment:
I think we should give an example of overriding a component that has the 'overrideStyleParams'.
Maybe we should change this example to theme the Badge component? or to implement the overrideStyleParams in the button component.  WDYT?

Reply:
sounds like a good idea

Reply:
Yeah 👍